src/main/java/org/octri/messaging/email/LoggingEmailDeliveryStrategy.java

```diff
@@ -2,8 +2,13 @@
 
 import java.util.Optional;
 
+import org.octri.messaging.autoconfig.EmailProperties;
+import org.octri.messaging.exception.UnsuccessfulDeliveryException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.util.Assert;
+
+import io.micrometer.common.util.StringUtils;
```

Comment:
Is this the StringUtils you meant to use? Sometimes I am intending to use Apache's StringUtils and accidentally import this one instead.